arangod/Cluster/ServerState.cpp

```diff
@@ -478,18 +480,14 @@ std::string ServerState::getPersistedId() {
   FATAL_ERROR_EXIT();
 }
 
-static constexpr char const* currentServersRegisteredPref =
-    "/Current/ServersRegistered/";
-
 /// @brief check equality of engines with other registered servers
 bool ServerState::checkEngineEquality(AgencyComm& comm) {
   std::string engineName = EngineSelectorFeature::engineName();
 
-  AgencyCommResult result = comm.getValues(currentServersRegisteredPref);
+  AgencyCommResult result = comm.getValues(concatPath({CURRENT, SERVERS_REGISTERED}));
```

Comment:
The assembled path does not end with a `/` after the change. I guess that isn't a problem, just wanted to be sure this is not an oversight.

Reply:
It is not an oversight; yet I will look & test again that this does not cause a problem.